tests/integration_tests/test_client_creation.py

```
@@ -212,33 +183,25 @@ def test_client_creation_and_collection_management(self, db_client):
         preview_all = collection.peek(limit=10)
         assert len(preview_all["ids"]) == 3  # All 3 items
-        print(
-            f"\n✅ collection.peek(limit=10) returned {len(preview_all['ids'])} items"
-        )
+        print(f"\n✅ collection.peek(limit=10) returned {len(preview_all['ids'])} items")
 
         # Clean up: delete all test collections
         try:
             db_client.delete_collection(test_collection_name)
             print(f"   Cleaned up collection: {test_collection_name}")
         except Exception as cleanup_error:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)